app/_hub/kong-inc/opentelemetry/_index.md

```diff
@@ -98,6 +98,15 @@ params:
       datatype: number
       description: |
         The timeout, in milliseconds, for reading the response from the OTLP server.
+    - name: header_type
+      required: true
+      default: preserve
```

Comment:
How is it `required: true` if it has a default value?

Reply:
Within the schema of the plugin itself, it's technically marked as `required`:

https://github.com/Kong/kong/pull/10620/files#diff-8d295aaa72ee8a0a18dcb8010b0a73ac36a92e1e825b3202b1d1736d07d7e514R53

In addition, this follows the same format that is currently set for Zipkin plugin which also utilizes this propagation feature:

https://github.com/Kong/docs.konghq.com/blob/main/app/_hub/kong-inc/zipkin/_index.md?plain=1#L134

So I was following the previously established pattern with the documentation. If this needs to be updated please let me know!

Reply:
I changed required to false in the plugin's schema, you can update it here as well, so it will make more sense